src/hash_mergers/answer.rb

```diff
@@ -9,33 +9,19 @@ class Answer < HashMerger
       private
 
       def hash_array(*a)
-        default_array(*a).concat super
+        super.unshift(domain_answers)
       end
 
-      def default_array(groups:, node:)
-        [default_hash(:domain)].tap do |x|
-          x.push(default_hash(:group)) if groups
-          if node == 'local'
-            x.push(default_hash(:local))
-          elsif node
-            x.push(default_hash(:node))
-          end
-        end
-      end
-
-      def default_hash(section)
-        section_default(section).each_with_object({}) do |(key, value), memo|
-          memo[key] = value.default unless value.default.nil?
-        end
+      def domain_answers
+        loader.flattened_configure_section(:domain)
+              .reject { |_k, value| value.default.nil? }
+              .map { |key, value| [key, value.default] }
+              .to_h
```

Comment:
Rather than `map`ing and then `to_h`ing when you want a hash with only the values changed from an existing hash, you can instead use [`transform_values`](http://devdocs.io/ruby~2.4/hash#method-i-transform_values) which will let you do this in a single line.

Reply:
Thanks for pointing that out. However, this line of code has been removed in a latter commit.